source/docs/articles/drupal.md

```diff
@@ -1,6 +1,43 @@
 ---
 title: Drupal
-description: All about Drupal, on Pantheon.
+description: All about Drupal on Pantheon.
 category: drupal
-draft: true
+keywords: drupal, sites, pantheon, upstream
 ---
+[Drupal](https://www.drupal.org) is an open source content management platform powering millions of websites and applications. It’s built, used, and supported by an active and diverse community of people around the world.
+
+Pantheon is the only hosting platform hyper-optimized for both Drupal and WordPress. Because we optimize our entire infrastructure for websites we deliver the best [performance available](http://getlevelten.com/blog/kyle-taylor/best-bang-no-bucks-comparison-free-drupal-hosting-platforms).
```

Comment:
Drupal sites on Pantheon run an optimized version of [Drupal](link to drops-7), based on [Pressflow](link to pressflow-7), with an additional [Pantheon API module](link to article). When new Drupal 7 sites are created, the platform clones the drops-7 repository as a part of its process, and sets it as a remote to track, so that you can [apply core updates](link to article) with a single click. When existing Drupal sites are imported, our system replaces core with drops-7.